examples/artifacts/plot_detect_riemannian_potato_EEG.py

```diff
@@ -19,6 +19,7 @@
 from matplotlib import pyplot as plt
 from matplotlib.colors import to_rgb
 from matplotlib.animation import FuncAnimation
+from IPython.display import HTML
```

Comment:
I would not make a hard dep on IPython on examples. It should not be necessary for sphinx gallery

mne has animations but no import of IPython

See for example this page: https://mne.tools/stable/auto_examples/visualization/evoked_topomap.html?highlight=animation#animating-the-topomap

Reply:
Ok, I see. It requires a consequent refactoring...

Reply:
Why IPython can not be added only in dependencies for documentation? @agramfort

Reply:
What about this modification?
```
try:
    from IPython.display import HTML
    plt.rcParams["animation.embed_limit"] = 10
    HTML(potato.to_jshtml(fps=5, default_mode='loop'))
except ImportError:
    raise ImportError("Install IPython to plot animation in documentation")
```

Reply:
my reasoning is that ipython is not a small dependency and if we can avoid it it's better. But I also hear that it's more work... PyRiemann is a super light package with minimal deps and I think we should fight for this. But here it's indeed just for the doc...

Reply:
Ok, you can merge if you are OK.